src/NzbDrone.Core/Organizer/FileNameBuilder.cs

```diff
@@ -267,6 +267,7 @@ private void AddArtistTokens(Dictionary<string, Func<TokenMatch, string>> tokenH
             tokenHandlers["{Artist Name}"] = m => artist.Name;
             tokenHandlers["{Artist CleanName}"] = m => CleanTitle(artist.Name);
             tokenHandlers["{Artist NameThe}"] = m => TitleThe(artist.Name);
+            tokenHandlers["{Artist Id}"] = m => artist.ForeignArtistId ?? string.Empty;
```

Comment:
It might be nice to have an indication that this is a MusicBrainz Id automatically appended. I didn't initially add that because I am not sure if other metadata sources are/will be supported.

Reply:
Agree, we should identify this as musicbrainz so that it doesn't get confused with the DB Id